robot.py

```diff
@@ -1,4 +1,4 @@
-import math
+import math  # noqa: I001

```

Comment:
Why has this been ignored?
```suggestion
import math
```